js/levels.js

```diff
@@ -14,7 +14,7 @@ function createFirstLevel()
     [door.none, door.minecraftEntry, door.minecraft, door.none, door.goal, door.none],
     [door.none, door.both, door.Forbidden, door.none, door.none, door.none],
     [door.none, door.sofa, door.outdoor, door.chessMate, door.texture, door.chessStale],
-    [door.none, door.yellowBoxes, PlayerStartsAt(door.black), door.green, door.banner, door.threeHeads],
+    [door.none, door.art, PlayerStartsAt(door.black), door.green, door.banner, door.threeHeads],
```

Comment:
`yellowBoxes` are'nt implemented again so can you pls remove any other tile which is implemented twice?

Reply:
Yellow boxes tile not shown, that's why I removed it

Reply:
![image](https://user-images.githubusercontent.com/33794760/34883494-bfbce85c-f7df-11e7-801d-0ceec07af8ff.png)